src/main/java/net/minecraftforge/common/util/FakePlayerFactory.java

```diff
@@ -26,7 +26,7 @@ public static FakePlayer getMinecraft(WorldServer world)
         if (ret == null)
         {
             ret = FakePlayerFactory.get(world,  MINECRAFT);
-            MINECRAFT_PLAYER = new WeakReference(ret);
+            MINECRAFT_PLAYER = new WeakReference<FakePlayer>(ret);
```

Comment:
This must have been done on purpose, but for simplicities sake I have changed it as well (Unless there was a specific reason to NOT have a generic type here that I am unaware of)